var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -24,24 +24,47 @@
 ##############################################################################
 from spack import *
 
-
-class Scr(Package):
+class Scr(CMakePackage):
     """SCR caches checkpoint data in storage on the compute nodes of a
        Linux cluster to provide a fast, scalable checkpoint/restart
        capability for MPI codes"""
 
-    homepage = "https://computation.llnl.gov/project/scr/"
+    homepage = "http://computation.llnl.gov/projects/scalable-checkpoint-restart-for-mpi"
+
+    ## NOTE: scr-v1.1.8 is built with autotools and is not properly build here.
+    ## scr-v1.1.8 will be deprecated with the upcoming release of v1.2.0
+    # url      = "https://github.com/LLNL/scr/releases/download/v1.1.8/scr-1.1.8.tar.gz"
+    # version('1.1.8', '6a0f11ad18e27fcfc00a271ff587b06e')
+
+    version('master', git='https://github.com/llnl/scr.git', branch='master')
+
+    depends_on('pdsh')
+    depends_on('zlib')
+    depends_on('mpi')
+
+    variant('dtcmp', default=True, description="DTCMP")
+    depends_on('dtcmp', when="+dtcmp")
+
+    ## YOGRT not yet in spack
+    # variant('yogrt', default=True, description="Lib YOGRT")
+    # depends_on('yogrt', when="+yogrt")
+
+    ## MySQL not yet in spack
```

Comment:
`mariadb` is, though.  Is it worth making `mysql` a vdep and making `mariadb` and (some future) `mysql` providers?